.github/workflows/deploy.yml

```diff
@@ -28,8 +27,8 @@ jobs:
       - name: Setup Hugo
         uses: peaceiris/actions-hugo@v2
         with:
-            hugo-version: 'latest'
-            extended: true
+          hugo-version: "latest"
```

Comment:
not related to the PR, but just thinking if we should freeze the hugo version in case of major upgrade and breaking changes

Reply:
Yes good idea eric. Should I make that change in this Pr?

Reply:
done @erickhun